nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/model/InstanceType.java

```diff
@@ -8,7 +8,7 @@ public enum InstanceType {
 
     ACADEMIC_MONOGRAPH("AcademicMonograph"), ACADEMIC_CHAPTER("AcademicChapter"),
     ACADEMIC_ARTICLE("AcademicArticle"), ACADEMIC_LITERATURE_REVIEW("AcademicLiteratureReview"),
-    NON_CANDIDATE("NonCandidateInstanceType");
+    NON_CANDIDATE("NonCandidateInstanceType"), IMPORTED_CANDIDATE("ImportedCandidate");
```

Comment:
Må ha en enum for nvi-kandidater fra cristin, da de kan ha forskjellige InstanceTypes enn det som er våres nvi-service tillater.

Reply:
Like we discussed -> we should change the type of DbCandidate from enum InstanceType to String and save the types imported.

Reply:
If you want to split these changes into several PRs, just remove enum IMPORTED_CANDIDATE for now and don't set instanceType

Reply:
Det blir forskjellige pr, da vi bør migrere eksisterende kandidater

Reply:
Får også ikke til å ta denne vekk, da våre enums som må ha en verdi:
```
    private PublicationDetails mapToPublicationDetails(CandidateDao candidateDao) {

        return new PublicationDetails(candidateDao.candidate().publicationId(),
                                      candidateDao.candidate().publicationBucketUri(),
                                      candidateDao.candidate().instanceType().getValue(),
                                      mapToPublicationDate(candidateDao.candidate().publicationDate()),
                                      mapToCreators(candidateDao.candidate().creators()),
                                      candidateDao.candidate().channelType(),
                                      candidateDao.candidate().channelId(),
                                      candidateDao.candidate().level()
                                          .getValues().stream().findFirst().orElse(null));
    }

```